tests/python/gaia-ui-tests/gaiatest/apps/messages/regions/report.py

```diff
@@ -0,0 +1,38 @@
+import time
+
+from marionette_driver import expected, By, Wait
+from gaiatest.apps.base import Base
+from gaiatest.form_controls.header import GaiaHeader
+
+class Report(Base):
+
+    _time_sent_locator = (By.CSS_SELECTOR, 'span.sent-timestamp')
+    _time_received_locator = (By.CSS_SELECTOR, 'span.received-timestamp')
+    _header_locator = (By.ID, 'information-report-header')
+
+    _sender = (By.CSS_SELECTOR, 'a[data-name]')
+    _receiver = (By.CSS_SELECTOR, 'a.suggestion.js-contact-info')
+
+    def close_report(self):
```

Comment:
Nit: In the test we have:
```py
report.close_report()
```

In order to have less duplication, we can simply have:
```py
report.close()
```
The same information is provided: You know the report will be closed